ada_feeding/ada_feeding/behaviors/ros/tf.py

```diff
@@ -428,11 +349,41 @@ def update(self) -> py_trees.common.Status:
             if target_frame is not None:
                 # Compute the transform from the TF tree
                 try:
-                    transformed_msg = self.tf_buffer.transform(
-                        stamped_msg,
-                        target_frame,
-                        timeout,
-                    )
+                    if isinstance(stamped_msg, TwistStamped):
```

Comment:
Does TransformationRegistration not include TwistStamped? You can check with `tf_buffer.registration.print_me()`

If not, we should implement it as part of the static execution (i.e. in the global scope) of `helpers`, see http://wiki.ros.org/tf2/Tutorials/Transforming%20your%20own%20datatypes